thor/fork_config.go

```diff
@@ -41,7 +41,7 @@ func (fc ForkConfig) String() string {
 }
 
 // NoFork a special config without any forks.
-var NoFork = ForkConfig{
+var NoFork = &ForkConfig{
```

Comment:
Kept `var NoFork = ForkConfig{` here and use `&thor.NoFork` where you need to pass pointer. Also, the 'value copy' part can also be beneficial via `forkConfig := thor.NoFork`.

Reply:
Yeah think that makes sense 👍